### PR TITLE
Made it possible to force TiNy to use $appc ti

### PR DESCRIPTION
**Pull request by tujoworker**
In some cases, I prefer to use only $ti - but to force TiNy to use $appc ti - we now can set the preferAppc to true by adding "--appc"

### CODE REVIEW

**Comment by FokkeZB** [comment]
Thanks @tujoworker!

Could it also be that you'd want the reverse?

File: bin/cli.js [hunk 1]
```diff
@@ -129,6 +129,10 @@ else if (cmd === '-v' || cmd === '--version' || cmd === 'version') {
       }
 
     } catch (e) {}
+    
+    if (process.argv.indexOf('--appc') > 0) {
```
Comment (FokkeZB) [inline bin/cli.js]: Better use `--prefer-appc` like https://github.com/FokkeZB/appc-compat/blob/0554c2ac0287d181ac74cee9fe50f1d2b580ac0d/bin/cli#L23

**Comment by tujoworker** [comment]
Yes, '--prefer-ti' would be wonderful.

**Comment by FokkeZB** [comment]
OK, could you change it to that?

**Comment by tujoworker** [comment]
Now both "--prefer-appc" and "--prefer-ti" is supported. Hope it looks OK to You :)
The reason why I have to switch between them, is that "faster-titanium" does not work anymore with $appc (only with $ti). But once I have to build the app to an device, I'm forced to use $appc again.

**Comment by FokkeZB** [comment]
Forgive me my comment:

> Could it also be that you'd want the reverse?

That makes no sense (this way) since `false` is already the default for `--prefer-appc`. Could you get rid of that? I'll merge it then :)

**Comment by tujoworker** [comment]
Actually, I think it's fine like it is now. Cause, once there is defined a "appc-app-id" in the tiapp.xml, and I would prefer only $ti - then it would not be possible.

**Comment by FokkeZB** [comment]
True.

Could you document both options at https://github.com/FokkeZB/tn/blob/master/bin/cli.js#L173 ?

**Comment by tujoworker** [comment]
Sure, should have done this lately. Hope they where ok.

**Comment by FokkeZB** [comment]
Thanks!

**Pull request merged**